app/poros/incident_contact_info.rb

```diff
@@ -0,0 +1,34 @@
+class IncidentContactInfo
+  attr_reader :id, :title, :name, :email, :phone_number, :distance_miles, :distance_minutes, :city, :state
+
+  def initialize(attributes)
+    @id = attributes[:id]
+    @title = attributes[:title]
+    @name = attributes[:name]
+    @email = attributes[:email]
+    @phone_number = attributes[:phone_number]
+    @incident_location = attributes[:incident_location]
+    @contact_location = attributes[:contact_location]
+    @distance_miles = calculate_distance_miles
+    @distance_minutes = calculate_distance_minutes
+  end
+
+  def calculate_distance_miles
```

Comment:
I like how you were able to put place holders for our distance micro-service to mock our data coming in.